askap/imageaccess/ADIOSImage.h

```diff
@@ -98,6 +111,14 @@ template <class T> class ADIOSImage : public casacore::ImageInterface<T>
   casacore::LatticeRegion* regionPtr_p;
   casacore::uInt row_p;
   casacore::String config;
+  // PJE - not certain how best to store the adios configuration besides
```

Comment:
There is an AdiosStMan constructor that takes a config filename (`Adios2StMan(MPI_Comm mpiComm, std::string configFile, from_config_t);`). Alternatively, we could pass the parset down and create our own subset of config parameters. This is more complicated but might be more consistent with the Askapsoft way of doing things.

Reply:
Nevermind, saw your changes below, disregard the first statement.